### ISSUE TITLE
Is is possible to change theme in real time

### ISSUE THREAD

**Issue by boxtree** [Zodiase/meteor-mdl#51]
Is it possible to change the theme (colors) in real time on a running site?

The zodiase-mdl.json file has the default theme colors, I would like to have the MDL color selection wheel on my site (or similar) and then allow a user pick their own color theme.

Thanks

**Comment by Zodiase** [comment]
TL;DR: No out of box support from this package.

Well... Let's look at it this way: you want different users to see different themes, which means there has to be a mechanism for generating the theme stylesheet on the fly, based on whatever criteria you want. In other words, you need something at the client side to request for a theme stylesheet after the user is logged in, and at server side a nodejs program to compose the stylesheet. To make theming easy, hopefully the nodejs program only needs to compose SASS code. But for the client to consume the stylesheet, the final data has to be CSS code. So somehow you have to use a SASS compiler to produce the final CSS code.

With all the above setup, you can configure theming variables in your dynamic SASS code then load the MDL theme SASS to produce CSS files with different MDL themes.

**Comment by Zodiase** [comment]
If you are only interested in the pre-built theme CSS files, use the undocumented hidden method `_mdlGetAsset('dist/material.amber-cyan.min.css')` on the server side to get the `amber-cyan` theme CSS file data. You still need some dynamic CSS loading on the client side and a server endpoint to serve the raw CSS data.
